conans/test/model/transitive_reqs_test.py

```diff
@@ -115,6 +115,7 @@ def setUp(self):
 
     def root(self, content):
         processed_profile = ProcessedProfile()
+        self.loader.cached_conanfiles = {}
```

Comment:
Yes, it is needed. If you remove it, some test fail, because they call ``root()`` two times, and reuse the ``ConanFileLoader`` declared in the ``setUp()``